test/test_transforms.py

```diff
@@ -225,22 +227,25 @@ def test_to_tensor(self, channels):
     def test_to_tensor_errors(self):
         height, width = 4, 4
         trans = transforms.ToTensor()
+        np_rng = np.random.RandomState(0)
```

Comment:
Let's remove this one too

Reply:
I think they come from a rather chaotic `git revert` :) should have checked before pushing though